### DISCUSSION TITLE
Move user account from one authentication method to another?

### DISCUSSION THREAD

**Discussion by cdauth** [outline/outline#5468] [Self Hosting] [answered]
I am hosting Outline along with Authentik as the authentication provider. I have recently understood that Outline can also work with just e-mail authentication (“Email magic link”), and since Authentik is a true pain to maintain, I want to turn it off and switch to e-mail-only authentication.

For new user accounts, this is not a problem, they can register and log in using e-mail. However, for existing user accounts that were created through Authentik, this does not work. While OIDC is enabled, attempting to use “Continue with Email” with an e-mail address that was registered with an Authentik account forwards to Authentik rather than sending an e-mail. When OIDC is disabled, it simply redirects to the login page again, and no e-mail is sent.

Is there any way to switch the existing user accounts to using e-mail authentication?

**Comment by tommoor** [comment]
You'll have to go into the postgres database and set the `enabled=false` on the row in the `authentication_providers` table for your OIDC provider.

**Reply by cdauth** [reply]
Thanks for the advice. In my `authentication_providers` table I have two rows, both are for `oidc` (my instance just has two users, and it seems like one authentication provider was created for each user's e-mail provider). Setting `enabled` to `false` on both did not solve the problem, I still get redirected to the front page when attempting to authenticate. Deleting both rows caused Outline to fail on startup with a message that I have to run migrations.

I have also tried to clear the `user_authentications` table. After that, I could successfully generate an authentication e-mail, but after clicking the magic link, Outline tells me that my account is suspended.

Any other hints?

**Reply by tommoor** [reply]
> I have also tried to clear the user_authentications table. After that, I could successfully generate an authentication e-mail, but after clicking the magic link, Outline tells me that my account is suspended.

This should certainly work, perhaps the account _is_ suspended

**Comment by cdauth** [comment, marked as answer]
It seems that the solution for me was to clear the `user_authentications` table. Afterwards I could authenticate using e-mail, but I was told that my account was suspended. This error disappeared after a few hours (and possibly a few restarts), so it works now.
